network-area-diagram/src/test/java/com/powsybl/nad/layout/LayoutWithFixedTextNodePositionsTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.nad.layout;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.powsybl.diagram.test.Networks;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.nad.build.iidm.NetworkGraphBuilder;
+import com.powsybl.nad.build.iidm.VoltageLevelFilter;
+import com.powsybl.nad.model.Graph;
+import com.powsybl.nad.model.Point;
+import com.powsybl.nad.model.TextPosition;
+
+/**
+ * @author Massimo Ferraro {@literal <massimo.ferraro at soft.it>}
+ */
+public class LayoutWithFixedTextNodePositionsTest {
```

Comment:
remove the public identifier (see the sonar cloud code smell)

Reply:
fixed